apps/platform_runner/test/end_to_end_test.exs

```diff
@@ -102,19 +102,28 @@ defmodule PlatformRunner.EndToEndTest do
     assert_async sleep: 1_000, max_tries: 30, debug: true do
       with {:ok, result} <-
              Prestige.query(session, "select * from e2e__csv order by single_letter") do
-        assert Prestige.Result.as_maps(result) == [
-                 %{"single_letter" => "a", "number" => "1"},
-                 %{"single_letter" => "b", "number" => "2"},
-                 %{"single_letter" => "c", "number" => "3"}
-               ]
+        assert Enum.member?(Prestige.Result.as_maps(result), %{
+                 "single_letter" => "a",
+                 "number" => "1"
+               })
```

Comment:
```elixir
assert %{"single_letter" => "a", "number" => "1"} in Prestige.Result.as_maps(result)
```

Reply:
I think this is a more readable way to assert the same thing.

Reply:
I agree but we got an error on this before from the formatter

Reply:
Try it, then run `mix format`. My guess is it was a line length issue.

Reply:
Ah, yep that was it